src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -15,45 +15,81 @@ import {
   FacetRefinement,
   NumericRefinement,
 } from '../../lib/utils/getRefinements';
-import { Connector, TransformItems } from '../../types';
+import { Connector, TransformItems, CreateURL } from '../../types';
 
-export type ConnectorRefinement = {
+export type CurrentRefinementsConnectorParamsRefinementType =
+  | 'facet'
+  | 'exclude'
+  | 'disjunctive'
+  | 'hierarchical'
+  | 'numeric'
+  | 'query'
+  | 'tag';
+
+export type CurrentRefinementsConnectorParamsRefinement = {
+  /**
+   * The attribute on which the refinement is applied.
+   */
   attribute: string;
-  type: string;
+
+  /**
+   * The type of the refinement.
+   */
+  type: CurrentRefinementsConnectorParamsRefinementType;
+
+  /**
+   * The raw value of the refinement.
+   */
   value: string | number;
+
+  /**
+   * The label of the refinement to display.
+   */
   label: string;
+
+  /**
+   * The value of the operator (only if applicable).
+   */
   operator?: string;
+
+  /**
+   * The number of found items (only if applicable).
+   */
   count?: number;
+
+  /**
+   * Whether the count is exhaustive (only if applicable).
+   */
   exhaustive?: boolean;
 };
 
-type ConnectorNumericRefinement = {
-  value: number;
-} & ConnectorRefinement;
-
-export type Item = {
+export type CurrentRefinementsConnectorParamsItem = {
+  /**
+   * The index name.
+   *
+   * @internal
```

Comment:
Is this `indexName` public?

Reply:
We use it internally to [create the React key](https://github.com/algolia/instantsearch.js/blob/56fcb8ccb185ce24d63ca8f5c8e5ae193836226b/src/components/CurrentRefinements/CurrentRefinements.tsx#L43). That's useful in a multi-index context—so I think we can make it public.